scripts/poverty-inequality-explorers/lis/lis_inequality_explorer.py

```diff
@@ -90,7 +90,6 @@
 
 processing_description = new_line.join(
     [
-        "The Luxembourg Income Study data is created from standardized household survey microdata available in their [LISSY platform](https://www.lisdatacenter.org/data-access/lissy/). The estimations follow the methodology available in LIS, Key Figures and DART platform.",
```

Comment:
Hi @lucasrodes! Is there a reason to remove this text? I feel the next paragraphs are closely related to it.